fili-core/src/test/groovy/com/yahoo/bard/webservice/logging/blocks/BardQueryInfoSpec.groovy

```diff
@@ -2,15 +2,21 @@
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
 package com.yahoo.bard.webservice.logging.blocks
 
-import com.yahoo.bard.webservice.web.ErrorMessageFormat
+import com.yahoo.bard.webservice.application.ObjectMappersSuite
+import com.yahoo.bard.webservice.logging.RequestLog
+
 import spock.lang.Specification
 import spock.lang.Unroll
 
+import java.util.concurrent.atomic.AtomicInteger
+
 class BardQueryInfoSpec extends Specification {
     BardQueryInfo bardQueryInfo
 
     def setup() {
         bardQueryInfo = BardQueryInfoUtils.initializeBardQueryInfo()
+        RequestLog.getId() // initialize RequestLog
```

Comment:
`RequestLog` is already initialized and populated in `BardQueryInfoUtils.initializeBardQueryInfo()`.